content/200-concepts/100-components/01-prisma-schema/index.mdx

```diff
@@ -136,6 +136,31 @@ Introspecting based on datasource defined in prisma/schema.prisma …
 Run prisma generate to generate Prisma Client.
 ```
 
+## Accessing environment variables from the schema
+
+Environment variables can be accessed using the `env` function:
+
+```prisma
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+In version 2.7.0 and later, you can expand variables using the format specified by [dotenv-expand](https://github.com/motdotla/dotenv-expand). This is useful when dealing with PaaS or similar products. such as Heroku):
+
+```
+# .env
+DATABASE_URL=postgresql://test:test@localhost:5432/test
+DATABASE_URL_WITH_SCHEMA=${DATABASE_URL}?schema=public
+```
```

Comment:
This is not a feature of the schema and `env()`, but the `.env` file (or the library we use to parse that file's content into env vars that are picked up by `env()`).

Reply:
That would suggest this is moved to the `Environment files` (nice name suggestion btw)  section instead as its not exclusive to the `env` function